code/game/objects/structures.dm

```diff
@@ -41,6 +41,14 @@
 
 	do_climb(usr)
 
+/obj/structure/attack_alien(mob/living/carbon/xenomorph/user)
+	. = ..()
+	do_climb(usr)
+
+/obj/structure/attack_hand(mob/living/carbon/human/user)
+	. = ..()
+	do_climb(usr)
```

Comment:
This should would benefit from a keybind instead.
Need to make modifications to the system to allow for this.
It also needs to use `user` instead of `usr`, and check intent.